server/springboot/src/main/java/com/styra/tickethub_springboot/dao/model/LatencyMeasuringHTTPClient.java

```diff
@@ -27,17 +29,23 @@
  */
 public class LatencyMeasuringHTTPClient extends OPAHTTPClient {
 
+    Logger logger;
```

Comment:
Define logger like: private static Logger logger = Logger.getLogger(LatencyMeasuringHTTPClient.class.getName());

Reply:
I saw people online saying to do that, but I don't see why it would be desirable for it to be static. Seems like that would stop it from being changed at runtime, which seems undesirable to me?

Reply:
Also, wouldn't making it static mean that every instance of the http client would share a single instance of the logger? That seems like the wrong thing to do to me.

Reply:
Yes, static means that each instance would use the same logger.  Due to the nature of the logger, that is what is desired.

Reply:
I think it would be totally reasonable for someone to want to instantiate 2 different OPA clients, each with a separate HTTPClient, and I think they might reasonably want them to have different loggers. If the logger was made static, that use case would be impossible. On the other hand, I don't see any downside to it not being static.

Reply:
Resolving this as discussion has moved to opa-java.